### PR TITLE
feat: ADD Projects Overview's infinite auto-scrolling carousel

### PR DESCRIPTION
**Pull request by ralphcajipe**
- Categories and the number of thesis titles for each
- An array to store all the placeholders for thesis posters
- Links to the Hall of Fame and Collections of Projects pages
- The CDN (Content Delivery Network) link for the scroll-carousel stylesheet
- The carousel that displays the thesis posters

The Infinite auto-scrolling carousel containing each one a stack of posters is **not yet in stacking style**  where the number of thesis posters are piled from behind as shown in our Figma. That is the term - stacking. I will be needing the help of @kylekce to take over the Tailwind CSS code or something equivalent. We should also consider the **fair treatment** where we have to randomize or shuffle on what poster will be displayed at the front - giving a chance for each to be shown in front. This can be done when the page reloads but it is better if it is done dynamically or without reloading the page with vanilla JS (plain JS).

Expected:
image

Current code with placeholder:
- Screenshot 1
image

- Screenshot 2
image

### CODE REVIEW

File: cs-expo-2023/app/projects/page.tsx [hunk 1]
```diff
@@ -1,11 +1,111 @@
-import React from 'react'
```
Comment (ralphcajipe) [inline cs-expo-2023/app/projects/page.tsx:1]: To the Reviewers:

If you need to preserve this import, feel free to do so. I think it was just my VS Code's intellisense that removed it.

**Review by nchlnnmrvnz — CHANGES_REQUESTED** [review]
- Scroll Carousel module not found

Other comments based on code structure:
- @rlme17 already created a component called ProjectsHeader.tsx. That component had hardcoded text, but you (or him) should revise it so it accepts two parameters of type string for the text: header & subheader (reusable when you start editing your page.tsx files for Projects Overview 1 and 2). That way, anyone can reuse that component and just call the component in your respective page.tsx files then supply the needed arguments to make the call successful.
- The carousel for the display of thesis posters uses a rectangle-shaped element where we put the picture. Ideally, since this rectangle element is reused in several parts of the design it should be made into a component. If it's a component, you can simply do the same thing I explained in the first bullet. You can call your component in your page.tsx.

So generally, other than the module not found error for scroll carousel, I am simply requiring you to refactor your code.

**Comment by ralphcajipe** [comment]
merge conflict was resolved.

**Comment by ralphcajipe** [comment]
@nchlnnmrvnz
I will close this PR in order to do a fresh PR due to the pruning of the branches. This will avoid any errors.

**Pull request closed without merging**